### ISSUE TITLE
Sphinx examples format issue in v1.7.1

### ISSUE THREAD

**Issue by steut001** [PyCQA/docformatter#225]
The following behavior has started happening in version 1.7.1, this did not happen in 1.7.0. The following code is in a docstring at the top of a python file that is used to create Sphinx gallery examples.

```
"""
===============================
Example of creating an example.
===============================
 ...
"""
```

is being reformatted to this, which is not desired
```
"""=============================== Example of creating an example.
===============================
 ...
"""
```

Was there a new rule that was implemented causing this problem that didn't previously exist. I'd rather not have to exclude the examples folder from the docformatter checks.

**Comment by weibullguy** [comment]
I can't reproduce this.  Could you provide a link to the file so I can use it to troubleshoot?

**Comment by steut001** [comment]
If that doesn't work, here is the entire contents of the file.

```python
"""
===============================
Example of creating an example.
===============================

.. currentmodule:: my_project

In this example, we illustrate how to create an example.
"""
print(__doc__)

# Authors: Tara Wilson

# %%
# Example of printing an environment variable
#############################################
#
# Explain what this example is doing and anything the user should know.
#

import os

path = os.getenv("PATH", "hello")
print(path)
```

**Comment by weibullguy** [comment]
OK.  Now I can reproduce the problem.  Thanks for the feedback, I'll get on this tonight.

**Comment by weibullguy** [comment]
@steut001 this should be fixed in v1.7.2-rc4.
